### PR TITLE
Split the install and start methods

### PR DESCRIPTION
**Pull request by daleharvey**
So users can programatically install which versions they require

### CODE REVIEW

**Comment by daleharvey** [comment]
Should really be testing this properly, looking at the test setup now

File: install.js [hunk 1]
```diff
@@ -161,3 +155,15 @@ function getIeDriverPlatform() {
...
+
+exports.install = function(callback) {
+  async.series([
+    setup,
+    download,
+    chmodChromeDr.bind(null, conf.chromeDr.path)
+  ], callback);
+}
+
+if (require.main === module) {
```
Comment (vvo) [inline install.js]: @daleharvey what does this does

Comment (daleharvey) [inline install.js]: If you run it from the command line, it calls that function, if its required then nothing happens

**Comment by daleharvey** [comment]
Closed in favour of v3

**Pull request closed without merging**